python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -35,9 +35,9 @@
 import subprocess
 import logging
 
+import lsst.dax.apdb as daxApdb
```

Comment:
Minor, but should this be after `ctrl.mpexec` imports?

Reply:
I'm pretty sure `dax.apdb` is much earlier in the dependency tree than `ctrl.mpexec`...

Reply:
Hmm, [it's actually undefined](https://peps.python.org/pep-0008/#imports). Fine.